bin/interactions/fun/xkcd.dart

```diff
@@ -36,20 +37,22 @@ class FunXkcdInteractions {
   }
 
   EmbedBuilder xkcdEmbed(SlashCommandInteractionEvent event, String title,
-      String desc, String imgUrl, String footerText) {
+      String desc, String imgUrl) {
     return EmbedBuilder()
       ..title = title
       ..description = desc
       ..color = DiscordColor.sapGreen
+      ..timestamp = DateTime.now()
       ..imageUrl = imgUrl
       ..addFooter((footer) {
-        footer.text = footerText;
+        footer.text = 'Requested by ${event.interaction.userAuthor?.username}';
+        ;
```

Comment:
has this semicolon been added by mistake?

Reply:
Yes, my bad.